pom.xml

```diff
@@ -14,15 +14,15 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.3.RELEASE</version>
+        <version>2.7.9</version>

```

Comment:
can we try bumping to 2.7.10 as thats the latest one.

Reply:
updated